src/number/format-decimal.js

```diff
@@ -48,3 +54,17 @@ module.exports = function(value, localeData, options) {
 	return ret;
 
 };
+
+function formatExponentialDecimal( value) {
+	// Get a value like "1.25e-8" and find its values
+	value = parseFloat(value).toExponential();
+
+	var pieces = value.split( 'e-' ),
+		ret = pieces[0].replace( pieces[0].charAt(1), '' ),
```

Comment:
Was thinking some locales might use a comma instead, but checking the [ES5 spec][0], it specifies a period. Will change.

[0]: http://www.ecma-international.org/ecma-262/5.1/#sec-7.8.3